exec/java-exec/src/main/java/org/apache/drill/exec/ops/FragmentContext.java

```diff
@@ -229,7 +229,7 @@ public DrillbitContext getDrillbitContext() {
     return context;
   }
 
-  public SchemaPlus getRootSchema() {
+  public SchemaPlus getFullRootSchema() {
```

Comment:
Comment to explain what a "full root schema" is? Apparently, this is both the plugin config name and workspace combined?